### PR TITLE
Installation single ai instance Ubuntu only

### PR DESCRIPTION
**Pull request by anetroger**
Reviewing the new installation page (ubuntu only) rewritten as topics

### CODE REVIEW

File: src/docs/_sidebar.md [hunk 1]
```diff
@@ -50,10 +50,16 @@
   - [For Data Scientists](/getting-started/engine-developer/)
     <!-- - [For Solution Architects](/getting-started/ml-integrator/) -->
   - [For MLOps Engineers](/getting-started/mlops-engineer/)
-- [Self-Hosted aiWARE](/aiware/install/installation-guide-overview.md)
-  - [Install Single Machine](/aiware/install/install.md)
-  - [Install Cluster](/aiware/install/cluster.md)
-  - [Validate Install](/aiware/install/validate.md)
+- [User-managed aiWARE](/aiware/install/installation-guide-overview.md)
+  - [Installation single aiWARE instance](/aiware/install/install.md)
```
Comment (myoungerman) [inline src/docs/_sidebar.md:54]: Should this be "Install" instead?

File: src/docs/_sidebar.md [hunk 2]
```diff
@@ -50,10 +50,16 @@
...
+    - [System requirements](/aiware/install/sysreq_single_ai_instance.md) 
+    - [Get UUID](aiware/install/get_uuid.md)
+    - [Set up environment](aiware/install/env_single_ai_instance.md)
+    - [Install aiWARE](aiware/install/install_single_ai_instance.md)
+    - [Add SSL certificates](aiware/install/add_ssl_certificates.md)
+    - [Log in Veritone](aiware/install/login_veritone.md)
+  - [Installation cluster](/aiware/install/cluster.md)
```
Comment (myoungerman) [inline src/docs/_sidebar.md:61]: Same question here

File: src/docs/aiware/install/add_ssl_certificates.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+1. In the terminal, change to the folder where certificates are located, and then type the following commands:
+
+    ```
+    cd /opt/aiware/haproxy/certs
+    curl https://get.aiware.com/anywhere/certs/server.pem -sLO
+    curl https://get.aiware.com/anywhere/certs/server.pem.key -sLO
+    ```
+
+2. Restart the aiware-​​HAProxy container. Type the following command:
```
Comment (myoungerman) [inline src/docs/aiware/install/add_ssl_certificates.md:22]: Does this command restart the container? Right now, it sounds like these are 2 separate instructions.

File: src/docs/aiware/install/add_ssl_certificates.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+# Add SSL certificates to the installation
+
+Install SSL certificates for the Veritone apps and restart the container.
```
Comment (myoungerman) [inline src/docs/aiware/install/add_ssl_certificates.md:3]: This intro sentence seems abrupt.

File: src/docs/aiware/install/ai_install_hub_core.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+# ai hub install core command
+
+When you run the `ai hub install core` command, you need to monitor the progress of background processes and make sure that the output information that appears in the terminal is saved.
```
Comment (myoungerman) [inline src/docs/aiware/install/ai_install_hub_core.md:3]: The last part of this sentence is passive voice--if the user is the one saving the info, I'd change it to active voice.

File: src/docs/aiware/install/ai_install_hub_core.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+
+The `ai hub install core` command does the following:
+
+* Installation of Core components as a background process
+* Update of Docker containers as a background process
+* Output of command lines in the terminal
```
Comment (myoungerman) [inline src/docs/aiware/install/ai_install_hub_core.md:9]: I think the tense of these verbs should change, so "installation" becomes "install", "update" becomes "updates", etc.

File: src/docs/aiware/install/ai_install_hub_core.md [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+
+The installation command outputs several lines, among which a URL with a token and a list of mapping of IP addresses to URLs.
```
Comment (myoungerman) [inline src/docs/aiware/install/ai_install_hub_core.md:11]: Replace "among which" with "including"

File: src/docs/aiware/install/ai_install_hub_core.md [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+
+To access the Veritone application, you need the URL. When the command outputs message lines, you must copy the URL and save it in a text file.
+
+To map the IP addresses to URLs, you also need to add the list to your system configuration file. 
+
+It is very important that you do not try to access the URL that is in the output of the terminal command before the installation is complete.
```
Comment (myoungerman) [inline src/docs/aiware/install/ai_install_hub_core.md:17]: Delete "It is very important that you" and replace it with >! to add a warning banner

File: src/docs/_sidebar.md (hunk 1 above)
Comment (anetroger) [reply src/docs/_sidebar.md:54]: you're suggestion is correct

File: src/docs/_sidebar.md [hunk 9]
```diff
@@ -50,10 +50,16 @@
...
+  - [Installation validation](/aiware/install/validate.md)
   - [Cluster Management Guide](/aiware/manage/)
     - [Air Gapped Install](/aiware/install/airgap/)
```
Comment (lulujo) [inline src/docs/_sidebar.md:64]: Should be "Air-gapped"

File: src/docs/aiware/install/ai_install_hub_core.md (hunk 5 above)
Comment (lulujo) [reply src/docs/aiware/install/ai_install_hub_core.md:3]: Suggest changing "that the output" to "the output" as there's another instance of "that" shortly after and this sounds repetitive.

File: src/docs/aiware/install/ai_install_hub_core.md (hunk 6 above)
Comment (lulujo) [reply src/docs/aiware/install/ai_install_hub_core.md:9]: +1

File: src/docs/aiware/install/ai_install_hub_core.md (hunk 7 above)
Comment (lulujo) [reply src/docs/aiware/install/ai_install_hub_core.md:11]: "list of mapping of" is grammatically incorrect. Maybe "a list of IP addresses mapped to URLS" ?

File: src/docs/aiware/install/env_single_ai_instance.md [hunk 10]
```diff
@@ -0,0 +1,41 @@
+# Set up the environment for aiWARE with the Core component
+
+For aiWARE with the Core component, set up environment variables and create a yaml file for the controller.
+
+>? Tip: You can create a text file with all command lines below, replace the UUID value, and then paste the command lines in the terminal.
+
+## Steps
+
+1. In the terminal, run the following commands:
+    * To change to root user:
```
Comment (lulujo) [inline src/docs/aiware/install/env_single_ai_instance.md:10]: add "the" before root user

File: src/docs/aiware/install/env_single_ai_instance.md [hunk 11]
```diff
@@ -0,0 +1,41 @@
+# Set up the environment for aiWARE with the Core component
+
+For aiWARE with the Core component, set up environment variables and create a yaml file for the controller.
```
Comment (lulujo) [inline src/docs/aiware/install/env_single_ai_instance.md:3]: YAML is actually capitalized - we probably should check this throughout all the docs.

File: src/docs/aiware/install/add_ssl_certificates.md (hunk 4 above)
Comment (anetroger) [reply src/docs/aiware/install/add_ssl_certificates.md:3]: I see your point, I've rewritten.

File: src/docs/aiware/install/ai_install_hub_core.md (hunk 5 above)
Comment (anetroger) [reply src/docs/aiware/install/ai_install_hub_core.md:3]: Agree, thanks for the suggestion.

File: src/docs/aiware/install/ai_install_hub_core.md (hunk 6 above)
Comment (anetroger) [reply src/docs/aiware/install/ai_install_hub_core.md:9]: I have rewritten the list as you suggested.

File: src/docs/_sidebar.md (hunk 9 above)
Comment (anetroger) [reply src/docs/_sidebar.md:64]: oh ok - that's why there is that 404 :) Who knows how log it has been there. I'll correct it.

Comment (anetroger) [reply src/docs/_sidebar.md:64]: I haven't touched this one yet, but I can correct it and also correct the 404 that we have.

File: src/docs/aiware/install/env_single_ai_instance.md (hunk 10 above)
Comment (anetroger) [reply src/docs/aiware/install/env_single_ai_instance.md:10]: done

File: src/docs/aiware/install/env_single_ai_instance.md (hunk 11 above)
Comment (anetroger) [reply src/docs/aiware/install/env_single_ai_instance.md:3]: Great point! Definitely an entry for the terminology system. I'll check all my files in the PR.

[the remark this answers is not in the document]
Comment (anetroger) [reply src/docs/aiware/install/env_single_ai_instance.md:3]: Absolutely right.

File: src/docs/aiware/install/env_single_ai_instance.md [hunk 12]
```diff
@@ -0,0 +1,41 @@
...
+        export AIWARE_MODE=redis,db,nsq,es,api,lb,minio,engine,automate,controller
+        export AIWARE_HOST_EXPIRE=false
+        export AIWARE_DOMAIN_NAME=dev-local.aiware.run
+        export AIWARE_INIT_TOKEN=uuid
+        ```
+        where uuid is the UUID that Veritone provided or you generated.
+
+    * To create a yaml file:
+
+        ?> Tip: Make sure there is exactly 1 space before the word default and 2 spaces before the words url and token.
```
Comment (lulujo) [inline src/docs/aiware/install/env_single_ai_instance.md:29]: Possibly put default, url, and token in quotes. Do we have a standard for this in the style guide?

File: src/docs/aiware/install/get_uuid.md [hunk 13]
```diff
@@ -0,0 +1,15 @@
+# Get a UUID
+
+UUID is a custom token that you can use to protect your installation. If Veritone didn’t provide a UUID, you can generate one.
+
+!> Warning: The installation database stores the first UUID that you write (export) to aiWARE environment variables. If you generate and export a new UUID, then you need to remove the first installation and then restart the installation from scratch. If you generated and exported multiple UUIDs, you might see 403 errors or other connection problems in the output of the installation command in the terminal.
```
Comment (lulujo) [inline src/docs/aiware/install/get_uuid.md:5]: "in" not "to" - and is it correct that we write the UUID to multiple variables? Or just one?

Comment (lulujo) [reply src/docs/aiware/install/get_uuid.md:5]: I also suggest changing "then you need to" to "you need to" as "then" is used again in that sentence.

File: src/docs/aiware/install/install_single_ai_instance.md [hunk 14]
```diff
@@ -0,0 +1,116 @@
...
+2. Type the following command:
+
+    ```
+    ai hub install core
+    ```
+
+
+    >! Warnings
+
+    * `ai hub install core `triggers a background process that you need to monitor and also outputs message lines in the terminal that you need to copy and save.
```
Comment (lulujo) [inline src/docs/aiware/install/install_single_ai_instance.md:59]: suggest a comma after "monitor" as this is a long sentence

File: src/docs/aiware/install/env_single_ai_instance.md (hunk 12 above)
Comment (anetroger) [reply src/docs/aiware/install/env_single_ai_instance.md:29]: Ok. Not yet, I'll work on it.

File: src/docs/aiware/install/install_single_ai_instance.md (hunk 14 above)
Comment (anetroger) [reply src/docs/aiware/install/install_single_ai_instance.md:59]: done

**Review by lulujo — APPROVED** [review]

**Pull request merged**